doc/start/os-recommendations.rst

```diff
@@ -22,6 +22,9 @@ Linux Kernel
   *at least* 4.19-based "longterm maintenance" kernel series.  If you can
   use a newer "stable" or "longterm maintenance" kernel series, do it.
 
+  For KRBD, a kernel of *at least* version 5.4 is the minimum necessary for
```

Comment:
K is assumed since we are in the "Ceph Kernel Client" section:
```suggestion
  For RBD, a kernel of *at least* version 5.4 is the minimum necessary for
```

Reply:
It is so revised.